cmd/messages_text.go

```diff
@@ -80,21 +80,34 @@ func (mt *MessagesText) reset() {
 	mt.Highlight()
 }
 
+// Region tags are square brackets that contain a region ID in double quotes
+// https://pkg.go.dev/github.com/rivo/tview#hdr-Regions_and_Highlights
+func (mt *MessagesText) startRegion(msgID discord.MessageID) {
+	fmt.Fprintf(mt, `["%s"]`, msgID)
+}
+
+// Tags with no region ID ([""]) don't start new regions. They can therefore be used to mark the end of a region.
+func (mt *MessagesText) endRegion() {
+	fmt.Fprint(mt, `[""]`)
+}
+
 func (mt *MessagesText) createMessage(m discord.Message) {
+	mt.startRegion(m.ID)
+	defer fmt.Fprintln(mt)
+	defer mt.endRegion()
```

Comment:
Deferred functions are LIFO, so the `(*MessagesText).endRegion` is called before `fmt.Fprintln`. It might be better to just use `Fprintln` instead of deferring it for readability.